CHANGELOG.md

```diff
@@ -35,6 +38,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [FLASK] `Add snap_getBip32PublicKey` RPC method ([#15889](https://github.com/MetaMask/metamask-extension/pull/15889))
 - **[FLASK] BREAKING:** Remove deprecated snap_getBip44Entropy_* method ([#15889](https://github.com/MetaMask/metamask-extension/pull/15889))
 - [FLASK] Miscellaneous fixes to the Snaps platform ([#15889](https://github.com/MetaMask/metamask-extension/pull/15889))
+- [FLASK] `snaps-monorepo@0.22.2` ([#16161](https://github.com/MetaMask/metamask-extension/pull/16161))
+- [FLASK] `snaps-skunkworks@0.22.0` ([#16069](https://github.com/MetaMask/metamask-extension/pull/16069))
```

Comment:
```suggestion
```
This one can be removed as none of the added features have been enabled yet, they will be in separate PRs.